eng/Versions.props

```diff
@@ -23,7 +23,7 @@
     <FSCoreReleaseNotesVersion>$(FSMajorVersion)-$(FSMinorVersion)-$(FSBuildVersion)</FSCoreReleaseNotesVersion>
     <FSCoreVersionPrefix>$(FSMajorVersion).$(FSMinorVersion).$(FSBuildVersion)</FSCoreVersionPrefix>
     <FSCoreVersion>$(FSMajorVersion).$(FSMinorVersion).$(FSBuildVersion).$(FSRevisionVersion)</FSCoreVersion>
-    <FCSMajorVersion>38</FCSMajorVersion>
```

Comment:
> Why a major version bump, is this because of the change to type XmlDoc =? can this be done in a non-breaking way?

We've had this discussion before. We make absolutely no effort for FCS to be binary compat and now is not the time to start.  If we are going to make that effort, we must first do a complete API review and make more things internal. etc. etc.

The policy is: bump whenever we feel like it.

> Also note that version 38 is not released yet

There are a lot of integers.

Reply:
I will add to the documentation notes, and use 38 for now